Client/src/components/contanier/staff/Courses/AddCourse/index.js

```diff
@@ -1,11 +1,105 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import "./index.css";
+import LabeledInput from "../../../../common/LabeledInput";
+import Button from "../../../../common/Button";
 
-export default class Addstudent extends Component {
+export default class Addcourse extends Component {
   render() {
     return (
-      <div>
-        from add std.
-      </div>
-    )
+      <>
+        <h1 className="add-course-titel">
+          <span className="add-course-line ">Add Course</span>
+        </h1>
+        <div className="add-course">
+          <div className="left">
+            <LabeledInput
+              LabeledInputClassName="course-name"
+              id="courseName"
+              labelText="Course Name:"
+              labelClassName="course-label"
+              type="text"
+              name="courseName"
+              inputClassName="course-input"
+            />
+            <LabeledInput
+              LabeledInputClassName="numbers-of-students"
+              id="NO"
+              labelText="Numbers Of Students:"
+              labelClassName="course-label"
+              min="3"
+              max="30"
+              type="number"
+              name="numberOfStudent"
+              inputClassName="course-input"
+            />
+            <LabeledInput
+              LabeledInputClassName="description"
+              id="desc"
+              labelText="Description:"
+              labelClassName="course-label"
+              type="text"
+              name="description"
+              inputClassName="course-input"
+            />
+          </div>
+          <div className="right">
+            <LabeledInput
```

Comment:
@naremanhilles I have a small user experience suggestion, so feel free to ignore it 😅
it might be easier for the user if this input was a select / dropdown (and the options were 1, 2, 3, .. 11, 12) instead of a number.  Then you don't have to worry about decimal numbers, and the min/max can be frustrating for users (they have to "discover" them by entering bad input).  With a `<select>`, they can quickly see all of the valid options and they can use their mouse for it, they don't even have to type with their keyboard (and on my phone, I hate typing into forms!).

Reply:
ok i will fexed it